core/src/test/scala/com/raphtory/BaseCorrectnessTest.scala

```diff
@@ -66,14 +63,15 @@ abstract class BaseCorrectnessTest(
       test: TestQuery,
       graphEdges: Edges,
       resultsResource: String
-  ): IO[Unit] =
+  ): IO[Unit] = {
     Raphtory
       .newIOGraph()
       .use { g =>
-        g.load(Source(graphEdges, setGraphBuilder()))
+        g.load(CSVEdgeListSource(graphEdges))
```

Comment:
This significantly reduces the flexibility of the tests. Not sure I like it though realistically, I want to revisit how the tests work anyway.

Reply:
Probably fine for now.